### PR TITLE
Comparison view

### PR DESCRIPTION
**Pull request by teemutontti**
### Leaftel side-by-side
Now uses `leaflet-side-by-side` -plugin to view satellite images next to each other with a slider control. The functionality of this component will be accessible through the bottom/sidebar menu, which is currently in progress (see issue #52).

### CODE REVIEW

File: app/components/map/comparison/SideBySide.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      // @ts-expect-error Time is valid but not included in the type definition.
+      time: leftDate,
+      transparent: true,
+    });
+
+    const rightLayer = L.tileLayer.wms("/wms", {
+      attribution:
+        '&copy; <a href="https://dataspace.copernicus.eu/" target="_blank">Copernicus Data Space Ecosystem</a>',
+      layers: "TRUE_COLOR",
+      // @ts-expect-error Time is valid but not included in the type definition.
```
Comment (kuukkanen) [inline app/components/map/comparison/SideBySide.tsx:28]: I'm not sure this is needed anymore? Does the base leaflet library contain types for these as this was required only for the react version to suppress typescript errors.

**Review by kuukkanen — COMMENTED** [review]
Maybe this whole thing could be done in more "reacty" way? Possibly by creating your own component that takes the left and right layers as children or something akin to that.

File: app/components/map/comparison/SideBySide.tsx (hunk 1 above)
Comment (teemutontti) [reply app/components/map/comparison/SideBySide.tsx:28]: Without it still gives this in the editor: `Object literal may only specify known properties, and 'time' does not exist in type 'WMSOptions'.ts(2353)`

**Comment by kuukkanen** [comment]
Now you should be able to control the opacity of either left or right image.

**Pull request merged**